src/test/resources/org/broadinstitute/hellbender/tools/copynumber/model-segments-expected/multiple-sample-cr-ac-nac-tumor-1.modelBegin.af.param

```diff
@@ -1,6 +1,6 @@
 @HD	VN:1.6
 @RG	ID:GATKCopyNumber	SM:SM-74P4M-1
 PARAMETER_NAME	POSTERIOR_10	POSTERIOR_20	POSTERIOR_30	POSTERIOR_40	POSTERIOR_50	POSTERIOR_60	POSTERIOR_70	POSTERIOR_80	POSTERIOR_90
-MEAN_BIAS	1.093856	1.101608	1.112991	1.120081	1.124299	1.134289	1.142537	1.153566	1.164479
```

Comment:
Do you have an explanation for what is causing this jitter in the CNV math? Some of these numbers jitter by 25+% and i'm not sure i'm qualified to comment on what changed about them? Have you spoken to the CNV team about this?